app.rb

```diff
@@ -64,11 +69,13 @@ def check_selection(response)
     when '5'
       book = Book.create_book
       @books << book
-      puts 'Book created successfully' if @books.include?(book)
+      puts "\n\nBook created successfully\n\n" if @books.include?(book)
+      # json_book = JSON.generate({ title: book.title, author: book.author })
+      # File.write("books.json", "#{json_book},\n", mode: "a")
     when '6'
       rental = Rental.create_rental(@books, @people)
       @rentals << rental
-      puts "\n\nRental created successfully".colorize(color: :green).italic if @rentals.include?(rental)
+      puts "\n\nRental created successfully\n\n".colorize(color: :green).italic if @rentals.include?(rental)
```

Comment:
- [ ] I believe you installed the `colorize` gem on your machine without including it in the Gemfile. As a result, an error occurs because the gem cannot be found. Please add the `colorize` gem to your Gemfile 👍 
![image](https://user-images.githubusercontent.com/46839548/195734618-a54f8216-9d5c-48b2-bee8-4534c2bb2a7a.png)
Please do this so that the other aspects of your work can be reviewed 🙏